src/database/migrations/0016-alter-bulk_uploads-file_size.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE bulk_uploads
+  ADD COLUMN file_size INTEGER;
+
+COMMENT ON COLUMN bulk_uploads.source_key IS
```

Comment:
🍝 Copypasta alert! 🍝

```suggestion
COMMENT ON COLUMN bulk_uploads.file_size IS
```